cypress/integration/models/developer/applicationinventory/applicationinventory.ts

```diff
@@ -447,7 +448,7 @@ export class ApplicationInventory {
         selectItemsPerPage(100);
         this.selectApplication();
         clickByText(button, review);
-        cy.wait(8000);
+        cy.wait(8 * SEC);
```

Comment:
I guess here you need to wait for `0.8 * SEC`, you don't need to wait for 8 seconds, right?